server/serializers/subscriptions.go

```diff
@@ -102,20 +120,31 @@ type CreateSubscriptionBodyPayload struct {
 }
 
 type SubscriptionDetails struct {
-	MattermostUserID string `json:"mattermostUserID"`
-	ProjectName      string `json:"projectName"`
-	ProjectID        string `json:"projectID"`
-	OrganizationName string `json:"organizationName"`
-	EventType        string `json:"eventType"`
-	ServiceType      string `json:"serviceType"`
-	ChannelID        string `json:"channelID"`
-	ChannelName      string `json:"channelName"`
-	ChannelType      string `json:"channelType"`
-	SubscriptionID   string `json:"subscriptionID"`
-	CreatedBy        string `json:"createdBy"`
-	TargetBranch     string `json:"targetBranch"`
-	Repository       string `json:"repository"`
-	RepositoryName   string `json:"repositoryName"`
+	MattermostUserID                 string `json:"mattermostUserID"`
+	ProjectName                      string `json:"projectName"`
+	ProjectID                        string `json:"projectID"`
+	OrganizationName                 string `json:"organizationName"`
+	EventType                        string `json:"eventType"`
+	ServiceType                      string `json:"serviceType"`
+	ChannelID                        string `json:"channelID"`
+	ChannelName                      string `json:"channelName"`
+	ChannelType                      string `json:"channelType"`
+	SubscriptionID                   string `json:"subscriptionID"`
+	CreatedBy                        string `json:"createdBy"`
+	TargetBranch                     string `json:"targetBranch"`
+	Repository                       string `json:"repository"`
+	RepositoryName                   string `json:"repositoryName"`
+	PullRequestCreatedBy             string `json:"pullrequestCreatedBy"`
+	PullRequestReviewersContains     string `json:"pullrequestReviewersContains"`
```

Comment:
Same.

Reply:
these cannot be modified as these are the expected key names for AZD